content/en/tracing/trace_collection/custom_instrumentation/dotnet/otel.md

```diff
@@ -115,6 +115,39 @@ catch(Exception e)
 }
 ```
 
+## Adding span events
+
+You can add span events using the `AddEvent` API. This method requires an `ActivityEvent`constructed with the `name` parameter and optionally accepts `attributes` and `timestamp` parameters. The method creates a new span event with the specified properties and associates it with the corresponding span. 
+
+It requires an `ActivityEvent`, which is constructed with the following parameters:
```

Comment:
```suggestion
```
I do think this is a bit redundant, and it's clearer if we just leave it off.

Reply:
Thank you I missed it, added a new line with the version just in case, the team liked it so if is good with you there I am good to merge, thank you!